caas/kubernetes/provider/k8s.go

```diff
@@ -2152,16 +2181,20 @@ func operatorPod(podName, appName, agentPath, operatorImagePath, version string,
 	}
 
 	appTag := names.NewApplicationTag(appName)
-	podLabels := make(map[string]string)
+	podAnnotations := make(map[string]string)
 	for k, v := range tags {
-		podLabels[k] = v
+		podAnnotations[k] = v
 	}
-	podLabels[labelVersion] = version
+	podAnnotations[labelVersion] = version
+	podAnnotations["apparmor.security.beta.kubernetes.io/pod"] = "runtime/default"
```

Comment:
do we wanna use `podAnnotations` here as well?

Reply:
yup